app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
I want you go through the Slack thread where I have added review comments and fix the issue over here.

Reply:
I will definitely go through them right now and will do the needful to fix the issue.